openam-authentication/openam-auth-fr-oath/src/main/java/org/forgerock/openam/authentication/modules/fr/oath/AuthenticatorOATH.java

```diff
@@ -189,6 +194,12 @@ public void init(Subject subject, Map sharedState, Map options) {
         if (debug.messageEnabled()) {
             debug.message("OATH::init");
         }
+        Locale locale = getLoginLocale();
+        bundle = amCache.getResBundle(amAuthOATH, locale);
+        if (debug.messageEnabled()) {
+            debug.message("ForgeRock Authenticator locale=" + locale);
+        }
```

Comment:
It is recommended that you print debug messages before using the "locale" parameter.